pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [project]
 name = "drizzle"
 description = "A package for combining dithered images into a single image"
-requires-python = ">=3.8"
+requires-python = ">=3.9"
```

Comment:
What I am confused about is that this is basically a forced requirement since the code in this package still would run in Python 3.8. What is the point of this restriction? It is not like we have to do anything to continue supporting it in Python 3.8.

Reply:
that is true, however code using Python 3.8 will be using an older version of Numpy, so the results will diverge. We definitely can still keep Python 3.8 and test with it, it might just be more maintenance

Reply:
> however code using Python 3.8 will be using an older version of Numpy, so the results / API usage may diverge with new Numpy versions. We definitely can still keep Python 3.8 and test with it, it might just be more maintenance

I do not think this should be happening since all (3.8, 3.9, etc) tests are currently passing simultaneously. However, if you feel keeping 3.8 is more maintenance, then let's remove 3.8.